handlers/main.yml

```
@@ -1,5 +1,7 @@
 ---
 - name: Refresh font cache
-  command:
+  ansible.builtin.command:
     cmd: fc-cache
   listen: "refresh font cache"
+  register: cmd_out
+  changed_when: "cmd_out.rc == 0"
```

I don't think the quotes are necessary here:
```suggestion
  changed_when: cmd_out.rc == 0
```

---

I will test the changed_when but I believe it was failing when I didn't have that in quotes at first

---

See [here](https://github.com/cisagov/ansible-role-xfce-cool/pull/50#discussion_r1232327361).

---

Looks like it worked, I must have had something else wrong during the same run.  See d637078